tldw_Server_API/tests/e2e/fixtures.py

```
@@ -1676,7 +1676,7 @@ def verify_uploads_accessible(api_client, uploaded_items: List[Dict[str, Any]])
     @staticmethod
     def create_content_hash(content: str) -> str:
         """Create a hash of content for verification."""
-        return hashlib.md5(content.encode()).hexdigest()
+        return hashlib.md5(content.encode()).hexdigest()  # nosec B324
```

Ruff (0.15.1)

[error] Probable use of insecure hash functions in `hashlib`: `md5`

(S324)